pvlib/ivtools/sdm.py

```diff
@@ -120,25 +120,25 @@ def fit_cec_sam(celltype, v_mp, i_mp, v_oc, i_sc, alpha_sc, beta_voc,
 
 def fit_desoto(v_mp, i_mp, v_oc, i_sc, alpha_sc, beta_voc, cells_in_series,
                EgRef=1.121, dEgdT=-0.0002677, temp_ref=25, irrad_ref=1000,
-               root_kwargs={}):
+               init_guess=None, root_kwargs={}):
     """
     Calculates the parameters for the De Soto single diode model.
 
-    This procedure (described in [1]_) has the advantage of
-    using common specifications given by manufacturers in the
+    This procedure (described in [1]_) fits the De Soto model [2]_ using
+    common specifications given by manufacturers in the
     datasheets of PV modules.
 
-    The solution is found using the scipy.optimize.root() function,
-    with the corresponding default solver method 'hybr'.
-    No restriction is put on the fit variables, i.e. series
+    The solution is found using :py:func:`scipy.optimize.root`,
+    with the  default solver method 'hybr'.
```

Comment:
the world's smallest review suggestion... but I noticed it so...
```suggestion
    with the default solver method 'hybr'.
```